src/arviz_plots/backend/plotly/__init__.py

```diff
@@ -694,18 +774,34 @@ def xlabel(string, target, *, size=unset, color=unset, **artist_kws):
     )
 
 
-def xticks(ticks, labels, target, **artist_kws):
+def xticks(ticks, labels, target, *, rotation=unset, **artist_kws):
     """Interface to plotly for setting ticks and labels of the x axis."""
     if labels is None:
         labels = [str(label) for label in labels]
-    target.update_xaxes(tickmode="array", tickvals=ticks, ticktext=labels, **artist_kws)
+    kwargs = {
+        "tickmode": "array",
+        "tickvals": ticks,
+        "ticktext": labels,
+    }
+    if rotation is not unset:
+        kwargs["tickangle"] = rotation
```

Comment:
```suggestion
    kwargs = {
        "tickmode": "array",
        "tickvals": ticks,
        "ticktext": labels,
    }
    if rotation is not unset:
        kwargs["tickangle"] = -rotation
```

plotly seems to be interpreting the rotation in the oposite direction as the other two backends. In general I'd say use `"ticklabel": rotation` directly when defining kwargs, `unset` values are filtered out. But we can change the sign of `unset` so keep the if so we can change the sign and get the labels in the same direction in all backends. Same applies to `yticks`